packages/data-broker/src/services/dhis/pullers/EventsPuller.js

```diff
@@ -80,7 +79,7 @@ export class EventsPuller {
     const { code: programCode } = dataSource;
 
     // TODO remove `useDeprecatedApi` option as soon as `pullEventsForApi_Deprecated()` is deleted
-    const { useDeprecatedApi = true } = options;
+    const { useDeprecatedApi = false } = options;
```

Comment:
I would also love for us to delete the deprecated `dhis` event api, but we cannot currently because of this bug: https://github.com/beyondessential/tupaia-backlog/issues/818

The modern api uses event analytics under the hood, and will use `0` for empty values